src/main/java/us/kilroyrobotics/subsystems/Wrist.java

```diff
@@ -5,19 +5,107 @@
 package us.kilroyrobotics.subsystems;
 
 import static edu.wpi.first.units.Units.Degrees;
+import static edu.wpi.first.units.Units.Radians;
 
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.sim.SparkMaxSim;
+import com.revrobotics.spark.SparkAbsoluteEncoder;
+import com.revrobotics.spark.SparkBase.ControlType;
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkClosedLoopController;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.config.ClosedLoopConfig.FeedbackSensor;
+import com.revrobotics.spark.config.SparkBaseConfig.IdleMode;
+import com.revrobotics.spark.config.SparkMaxConfig;
 import edu.wpi.first.epilogue.Logged;
 import edu.wpi.first.math.geometry.Pose3d;
 import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.wpilibj.simulation.BatterySim;
+import edu.wpi.first.wpilibj.simulation.RoboRioSim;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import java.util.function.Supplier;
+import us.kilroyrobotics.Constants.CoralMechanismConstants;
+import us.kilroyrobotics.Constants.SimulationConstants;
 
 public class Wrist extends SubsystemBase {
+    private SparkMax m_wristMotor;
     private Supplier<Pose3d> getCarriagePose;
+    private boolean m_useAbsoluteEncoder;
+    private SparkAbsoluteEncoder m_absoluteEncoder;
+    private RelativeEncoder m_relativeEncoder;
```

Comment:
Do we need this object? We should only ever be looking at the absolute encoder

Reply:
Yes, for testing (Mr. Brown doesn't want to hook up the absolute encoder on just testing the motor working). In reality we will almost never use this, but it's there if we ever need it in the future.